### PR TITLE
tvOS + tvOS Example

### PR DESCRIPTION
**Pull request by mickeyreiss**
This PR takes #106 and adds an additional commit to include a tvOS sample integration.

Given the 3 :+1:s and my own personal testing, I'm more comfortable merging https://github.com/adrum/PureLayout/commit/57b53b6374e806ae4000a305b771297c16559fc6 now.

I'll leave this here for a day or two in case anyone has time to review, and then I'll merge/release.

Thanks for your patience, everybody!

***

cc/ @adrum @AvdLee @Mazyod

### CODE REVIEW

File: PureLayout/PureLayout.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -196,6 +229,15 @@
...
 
 /* Begin PBXFileReference section */
+		164C8B4E1C0D354E0007A6B1 /* Example-tvOS.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = "Example-tvOS.app"; sourceTree = BUILT_PRODUCTS_DIR; };
+		164C8B501C0D354E0007A6B1 /* TVApp.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TVApp.swift; sourceTree = "<group>"; };
+		164C8B551C0D354E0007A6B1 /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/Main.storyboard; sourceTree = "<group>"; };
+		164C8B571C0D354F0007A6B1 /* Assets.xcassets */ = {isa = PBXFileReference; lastKnownFileType = folder.assetcatalog; path = Assets.xcassets; sourceTree = "<group>"; };
+		164C8B591C0D354F0007A6B1 /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
+		164C8B711C0D36480007A6B1 /* PureLayout_Mac copy-Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = "PureLayout_Mac copy-Info.plist"; path = "/Users/mickeyreiss/personal_code/PureLayout/PureLayout/PureLayout_Mac copy-Info.plist"; sourceTree = "<absolute>"; };
```
Comment (blueraz0r) [inline PureLayout/PureLayout.xcodeproj/project.pbxproj]: hardcoded absolute path to your machine here

File: PureLayout/PureLayout.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -196,6 +229,15 @@
...
+		164C8B831C0D36500007A6B1 /* PureLayout.framework */ = {isa = PBXFileReference; explicitFileType = wrapper.framework; includeInIndex = 0; path = PureLayout.framework; sourceTree = BUILT_PRODUCTS_DIR; };
+		164C8B841C0D36500007A6B1 /* PureLayout_iOS copy-Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = "PureLayout_iOS copy-Info.plist"; path = "/Users/mickeyreiss/personal_code/PureLayout/PureLayout/PureLayout_iOS copy-Info.plist"; sourceTree = "<absolute>"; };
```
Comment (blueraz0r) [inline PureLayout/PureLayout.xcodeproj/project.pbxproj]: another hardcoded absolute path to your machine

**Comment by blueraz0r** [comment]
Very nice can't wait for this. What about modifying Travis CI `.travis.yml` file to build/run for tvOS? not sure if they support it yet

**Comment by mickeyreiss** [comment]
I've committed changes based on the review. Will merge once CI passes.

Thanks for the thorough review, @danie11am and @blueraz0r! As per the discussions in #80, feel free to reach out to me if you'd like to get more involved in helping maintain the library.

**Pull request merged**